src/sentry/lang/native/unreal.py

```diff
@@ -168,15 +194,23 @@ def merge_unreal_context_event(unreal_context, event, project):
 
     portable_callstack = runtime_prop.pop('portable_call_stack', None)
     if portable_callstack is not None:
-        images = get_path(event, 'debug_meta', 'images', filter=True, default=())
-        frames = parse_portable_callstack(portable_callstack, images)
-
-        if len(frames) > 0:
-            exception = get_path(event, 'exception', 'values', 0)
-            if exception:
-                # This property is required for correct behavior of symbolicator codepath.
-                exception['mechanism'] = {'type': 'unreal', 'handled': False, 'synthetic': True}
-            event['stacktrace'] = {'frames': frames}
+        if refactor_enabled:
+            set_path(event, 'contexts', 'unreal', 'type', value='unreal')
+            set_path(event, 'contexts', 'unreal', 'portable_call_stack',
+                     value=portable_callstack)
+            # TODO(markus): Add other stuff from extra here. Make sure trimming
```

Comment:
Note: Immediate follow-up should be to match this function with the implementation in `wip/symbolicator`. 

Apart from moving all extra into `contexts.unreal`, this also moves creating of user feedback to the processing step.